.travis.yml

```diff
@@ -45,7 +45,7 @@ matrix:
       if: type = cron
       before_install:
         - python -m pip install cython numpy scipy
-        - python -m pip install --no-build-isolation hypothesis matplotlib pytest pytest-cov pytest-xdist tqdm
+        - python -m pip install --no-build-isolation hypothesis matplotlib pytest pytest-cov pytest-xdist tqdm threadpoolctl
```

Comment:
I've lost track of why this was being added. Do we need to also add this to ppc64le?

Reply:
threadpoolctl is essential for the PR; we might have to promote it to "required package" because the transformation code will fail without it (I think)

Reply:
@IAlibay  should we also add to CHANGELOG under Changes that threadpoolctl is now required? There's also a wiki page where we list required packages (I think in the developer guide... not sure if this has been superceeded by anything in the User Guide).

Reply:
> @IAlibay should we also add to CHANGELOG under Changes that threadpoolctl is now required? 
👍🏽 

> There's also a wiki page where we list required packages (I think in the developer guide... not sure if this has been superceeded by anything in the User Guide).

I think we now link to the user guide (see here: https://github.com/MDAnalysis/mdanalysis/wiki/Guide-for-Developers).
We should definitely update the userguide install instructions (we probably need to have a blanket issue for this, since I think there are a few things that need ammending for 2.0? Tagging @lilyminium here.)

Reply:
Made https://github.com/MDAnalysis/UserGuide/issues/139 and https://github.com/MDAnalysis/UserGuide/issues/138.